CHANGELOG.md

```diff
@@ -3,6 +3,120 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the bundling scripts as the packages published by the [https://github.com/mui/material-ui](material-ui) repository (#3965) @flaviendelangle
+
+  The code structure and the bundling strategy have been modified to provide better isolation between components.
+  This brings tree shaking, allowing to reduce the bundle size. 🏋
```

Comment:
```suggestion
  This brings tree shaking to reduce the bundle size. 🏋
```

Reply:
@flaviendelangle I'm not sure, does the bundle size is already reduced, or will we need to take other actions to reduce it?

Reply:
How about?

```suggestion
  The bundle size is slightly reduced, but with tree shaking, the doors are open for significant gains in the future.
```